internal/v1/handler_post_compose_test.go

```diff
@@ -2921,6 +2925,16 @@ func TestComposeCustomizations(t *testing.T) {
 							Rhsm:     common.ToPtr(false),
 						},
 					},
+					Subscription: &composer.Subscription{
+						ActivationKey:       "",
+						BaseUrl:             "",
+						Insights:            true,
+						Rhc:                 common.ToPtr(false),
+						Organization:        "0",
+						ServerUrl:           "",
+						InsightsClientProxy: common.ToPtr(""),
+						TemplateUuid:        common.ToPtr(mocks.TemplateID),
+					},
```

Comment:
Is this hunk related?

Reply:
yes, this is needed for the test to pass